www/front_src/src/Resources/Details/tabs/Graph/index.tsx

```diff
@@ -40,7 +41,9 @@ const GraphTabContent = ({ details }: TabProps): JSX.Element => {
   const equalsService = equals('service');
   const equalsMetaService = equals('metaservice');
 
-  const isService = or(equalsService(type), equalsMetaService(type));
+  const isService =
+    or(equalsService(type), equalsMetaService(type)) ||
+    equals(type, ResourceType.anomalydetection);
```

Comment:
```suggestion
    equalsService(type) || equalsMetaService(type) ||
    equals(type, ResourceType.anomalydetection);
```

In order to clean the code, you can replace 'service' and 'metaservice' above by ResourceType.service and ResourceType.metaservice.
In the same way, you can refactor `equals(type, ResourceType.anomalydetection)` like equals functions above